### ISSUE TITLE
Fix social login

### ISSUE THREAD

**Issue by johnfelipe** [opengovfoundation/madison#971]
http://jhonfelipeurregomejia.com:2016/user/login

I want to integrate that login method

**Comment by krues8dr** [comment]
I'm not 100% sure that this works anymore - we haven't used it in a while.  In theory, you just need to supply your Facebook and Twitter API keys in the .env file:

```
'FB_CLIENT_ID' => 'Your Facebook API ID',    
'FB_CLIENT_SECRET' => 'Your Facebook API Secret',
'TW_CLIENT_ID' => 'Your Twitter API ID',
'TW_CLIENT_SECRET' => 'Your Twitter API Secret',
```

**Comment by krues8dr** [comment]
The Facebook API was just upgraded, so we probably need to do some extra work here.

**Comment by doshitan** [comment]
Closed in favor of #706.
